diff --git a/node_modules/react-native-maps/android/build.gradle b/node_modules/react-native-maps/android/build.gradle
index 067a913..6a190dd 100644
--- a/node_modules/react-native-maps/android/build.gradle
+++ b/node_modules/react-native-maps/android/build.gradle
@@ -64,7 +64,7 @@ dependencies {
   implementation 'com.facebook.react:react-native:+'
   implementation "com.google.android.gms:play-services-base:${safeExtGet('playServicesVersion', '18.2.0')}"
   implementation "com.google.android.gms:play-services-maps:${safeExtGet('playServicesVersion', '18.2.0')}"
-  implementation "com.google.android.gms:play-services-location:21.0.1"
+  implementation "com.google.android.gms:play-services-location:20.0.0"
   implementation 'com.google.maps.android:android-maps-utils:3.8.2'
   implementation "androidx.work:work-runtime:2.7.1"
 }
